test/fixtures/result_expected.js

```diff
@@ -15,7 +15,7 @@
       color: red;
     }
     smile:before {
-      content: "\000B";
+      content: "\\000B";
```

Comment:
These all needed their backslashes escaped so they will be created in memory as the string `\000B` rather than `[null character]B`